base/abstractarraymath.jl

```diff
@@ -2,8 +2,6 @@
 
  ## Basic functions ##
 
-isinteger(x::AbstractArray) = all(isinteger,x)
-isinteger{T<:Integer,n}(x::AbstractArray{T,n}) = true
```

Comment:
As opposed or in addition to decorating `all` with `@inbounds` as you suggest above? Thanks!